lib/biran/configurinator.rb

```diff
@@ -46,6 +46,16 @@ def create(name:, extension:, output_dir: nil)
       generated_file.save!
     end
 
+    def default_db_file?
+      config.fetch(:app, {}).fetch(:db_config, nil) != false
+    end
+
+    def config_files_to_generate
+      files = configuration.files_to_generate
+      return files unless default_db_file?
+      files.merge configuration.db_file_to_generate
```

Comment:
I still don't understand the separation here of treating the db file as different. This appears to all be just guarding against the idea of a missing db_config.yml file. The original defaults of generating both the vhost and the database.yml file was based on the idea that for the majority of our projects, that is what we want, so no additional config would be needed. Why change the defaults of a gem based on the existence of a file. Just because I have a db_config.yml file doesn't necessarily mean I want the database.yml file generated. But without changing anything in app_config, it appears this would happen based on presence of file.
So my question is, what problem is this change trying to solve?

Reply:
it is solving preventing Biran trying to generate the `database.yml` file and looking for `_database.yaml.erb` the presence of a file is only a condition, but there is some other checks that doesnt have to do with `db_config` there is actually no need for a db_config.yml file not sure if this is technical debt or what. But I still need to take out database out from default tasks and files to be generated.

Reply:
This is more easily solved by simply removing the database file from the default hash of files to be generated. Biran won't try to generate the file IF the config is set in app_config.yml. I still say this has nothing to do with preventing loading in ruby only.

Reply:
Tried that a lot. It seems to be be very coupled to the code so there is no way this will work ...

Reply:
```
This is more easily solved by simply removing the database file from the default hash of files to be generated. 
```

yeah that fixes the issue, but makes it incompatible with all our current installations